File: VB/CustomInfoInTooltips/Form1.vb
Imports System
Imports System.Windows.Forms

Namespace CustomInfoInTooltips

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs)
            ' TODO: This line of code loads data into the 'nwindDataSet.Products' table. You can move, or remove it, as needed.
            productsTableAdapter.Fill(nwindDataSet.Products)
            chartControl1.Series(0).CrosshairLabelPattern = "Unit price: {UnitPrice}" & Microsoft.VisualBasic.Constants.vbCrLf & "Units in stock: {UnitsInStock}" & Microsoft.VisualBasic.Constants.vbCrLf & "Quantity per unit: {QuantityPerUnit}"
        End Sub
    End Class
End Namespace

File: CS/CustomInfoInTooltips/Form1.cs
using System;
using System.Windows.Forms;
namespace CustomInfoInTooltips {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }
        private void Form1_Load(object sender, EventArgs e) {
            // TODO: This line of code loads data into the 'nwindDataSet.Products' table. You can move, or remove it, as needed.
            this.productsTableAdapter.Fill(this.nwindDataSet.Products);
            chartControl1.Series[0].CrosshairLabelPattern = "Unit price: {UnitPrice}\r\n" +
                                                            "Units in stock: {UnitsInStock}\r\n" +
                                                            "Quantity per unit: {QuantityPerUnit}";
        }
    }
}